src/main/java/at/hannibal2/skyhanni/features/event/diana/DianaAPI.kt

```diff
@@ -16,8 +16,8 @@ object DianaAPI {
 
     fun hasSpadeInHand() = InventoryUtils.itemInHandId == spade
 
-    private fun isRitualActive() = MayorElection.isPerkActive("Diana", "Mythological Ritual") ||
-        MayorElection.isPerkActive("Jerry", "Perkpocalypse") || SkyHanniMod.feature.event.diana.alwaysDiana
+    private fun isRitualActive() = Perks.MYTHOLOGICAL_RITUAL.isActive ||
+        Perks.PERKPOCALYPSE.isActive || SkyHanniMod.feature.event.diana.alwaysDiana
```

Comment:
the Perkocalpse shouldn't be used here. Since Perks.MYTHOLOGICAL_RITUAL. should be simply active when Jerry Diana is. See comment in data.Mayor